internal/generator/vector/vrl.go

```diff
@@ -0,0 +1,49 @@
+package vector
+
+import (
+	"fmt"
+	"strings"
+)
+
+var (
+	SkipEmpty = func(in []string) []string {
+		vals := []string{}
+		for _, v := range in {
+			if strings.TrimSpace(v) != "" {
+				vals = append(vals, v)
+			}
+		}
+		return vals
+	}
+	Paren = func(in string) string {
+		return fmt.Sprintf("(%s)", in)
+	}
+	ParenAll = func(in []string) []string {
+		if len(in) == 1 {
+			return in
```

Comment:
shouldn't this be Paren(in) ?

Reply:
if there is one element only, no need to parenthesize